### DISCUSSION TITLE
【灌水】前台小刘和coser的合照

### DISCUSSION THREAD

**Discussion by 5544707** [share121/inter-knot#2122] [其他]
![IMG_6193](https://github.com/user-attachments/assets/774bd776-ce86-47ca-b23e-86e1f2612094)
coser：@Chocolat_cos0

**Comment by wesdyuhj** [comment]
大伟哥！！

**Comment by wugan-bot156** [comment]
Coser的X账号:[しょこら](https://x.com/Chocolat_cos0)

**Comment by Xuanan0** [comment]
![5ddbb166-c860-4a76-b958-79470864baa4](https://github.com/user-attachments/assets/48f3dd74-34bc-4e0b-bcde-a2ab92a1c51f)

**Comment by jianjue1** [comment]
这都被屏蔽也太搞了

**Comment by WuLiao000000** [comment]
刚在推看到

**Comment by Kamisato100** [comment]
前台小刘也是出息了，居然能和椰羊合照了<img src="https://img-static.mihoyo.com/communityweb/upload/76cc94063d9c0efb2335e8487299cd0a.png" alt="银狼 吹泡泡" width="75" >
